person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] Please make the `rentals` accessible e.g `attr_accessor :rentals` so that you can set or assign values to the `rentals` array 👍